### DISCUSSION TITLE
10월 4일 주간회의

### DISCUSSION THREAD

**Discussion by euijinkk** [woowacourse/prolog#1051] [General]
현재 진행중인 작업에 대해서 공유할 내용이 있으면 코멘트로 공유해주세요~

**Comment by euijinkk** [comment]
로드맵 UI 작업 진행 중입니다.

[feat: 로드맵을 위한 하위 컴포넌트 구현](https://github.com/woowacourse/prolog/pull/1047)

**Comment by wonsss** [comment]
로드맵 UI - 사이드시트 컴포넌트 작업 진행하였습니다.

https://github.com/woowacourse/prolog/pull/1048

**Comment by wishoon** [comment]
### 레벨로그 댓글 
* PR Review 리뷰중 #1033

### 역량 로드맵 
* 도메인 설계 진행

![20220927_190244](https://user-images.githubusercontent.com/48710213/193746237-f48135ab-138e-4200-9386-8009f63662f7.jpg)

**Comment by al-bur** [comment]
## 임시 저장 기능
https://github.com/woowacourse/prolog/issues/765

<진행 상황>
학습로그 작성 페이지에서 임시 저장 기능 구현하였습니다.
학습로그 작성 페이지 입장시 임시 저장된 글이 있으면 불러오도록 구현하였습니다. (editor 본문은 내용이 바뀌지 않는 문제 해결 필요)


<특이 사항>
백엔드 로직 수정 요청 필요 (임시 저장 GET과 PUT 요청에 abillities가 포함되어야합니다)

**Comment by her0807** [comment]
## API설계 

금요일 3시에 설계 회의 하는 걸로

**Comment by her0807** [comment]
10월 24일 데드라인으로 하고.. 

링크는 없고, url 로 치고 들어가면 보여야 함

**Comment by Yboyu0u** [comment]
## Document -> NewDocument 사용하도록 수정 총 11개 중 9개 완료
~~Login~~
~~Mission~~
~~Profile~~
~~Comment~~
~~Member~~
~~MemberReaction~~
~~SessionMember~~
~~Tag~~
~~StudylogoverView~~
Studylog
StudylogRssFeed
